README.md

```diff
@@ -73,6 +73,20 @@ puck --extra-files config/*.yml
 
 The more advanced `:extra_files` feature, where the destination paths within the JAR file is specified as described below, is not available as a command line option.
 
+If your application is depending on another Jar you can merge it into the application Jar using the `:merge_archives` option:
+
+```ruby
+Puck::Jar.new(merge_archives: Dir['external.jar']).create
+```
+
+or using the command line:
+
+```
+puck --merge-archives external.jar
+```
+
+The more advanced `:merge_archives` feature, where the destination paths within the JAR file is specified as described below, is not available as a command line option.
```

Comment:
I know where you copied this from, but fix JAR -> Jar (you could fix it where you copied this from too).